packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -80,12 +87,29 @@ export class CarequalityManagementAPIImpl implements CarequalityManagementAPI {
       default:
         throw new Error("API mode not supported.");
     }
+    this.maxBackoff = options?.maxBackoff
+      ? dayjs.duration(options.maxBackoff, "seconds").asMilliseconds()
+      : DEFAULT_MAXIMUM_BACKOFF.asMilliseconds();
 
     this.api = axios.create({
       timeout: options?.timeout ?? DEFAULT_AXIOS_TIMEOUT.asMilliseconds(),
       baseURL: baseUrl,
       httpsAgent: this.httpsAgent,
     });
+
+    axiosRetry(this.api, {
+      retries: options?.retries ?? DEFAULT_MAX_RETRIES,
+      retryDelay: retryCount => {
+        const exponentialDelay = Math.pow(2, retryCount);
+        const jitter = Math.random();
+        const delayWithJitter = (exponentialDelay + jitter) * BASE_DELAY.asMilliseconds();
```

Comment:
Not a huge deal, but assuming `retryCount` starts at `1`, `BASE_DELAY` is not the initial delay - which one would assume that's what it means without reading the code.

This is because `Math.pow(2, retryCount)` will start at `2`. We could instead do `Math.pow(2, Math.max(0, retryCount - 1))`